### DISCUSSION TITLE
Supporting Ubuntu

### DISCUSSION THREAD

**Discussion by drweb86** [FarGroup/FarManager#550] [General]
Hi

I want to start topic about Ubuntu support. Want to know your current vision of it.

Recently Microsoft has started segregation of Belorussians by not allowing to purchase Windows, Store and Azure subscription. So I switched at home to Ubuntu. While I highly dislike its user experience and linux at all, I see some improvement. Finally sound cards are started to be supported, crashes reduced. So my comfort would be higher if at least something normal apart from Visual Studio code would be there like Far to which I'm sort of addicted.

**Comment by yegor-mialyk** [comment]
There is a fork for version 2 with Linux support https://github.com/elfmz/far2l

**Comment by drweb86** [comment]
Thanks, actually i tried Ubuntu for 2 months and returned back to Windows. Better to be seggregated but be able to work productively, then use it.

Ubuntu became better over 12 years since i last used Linux. No more crashes, better UI, graphic card support added, However, it is still bad for desktop use cases: games crash via windows emulators, external sound cards are not supported, browser always plays sound via monitor, square artefacts during play video and sound desync in VLC, no Far Manager of cause and normal software for photos, music DAW etc.

**Comment by bitekas** [comment]
You might want to consider my project: https://github.com/vmisoft/vscommander
It's a VSCode Far-replacement cross-platform extension
